resource-managers/kubernetes/integration-tests/README.md

```diff
@@ -269,3 +269,28 @@ to the wrapper scripts and using the wrapper scripts will simply set these appro
     <td></td>
   </tr>
 </table>
+
+# Running the Kubernetes Integration Tests with SBT
+
+You can use SBT in the same way to build image and run all K8s integration tests except Minikube-only ones.
+
+    build/sbt -Psparkr -Pkubernetes -Pkubernetes-integration-tests \
```

Comment:
Not directly related to this PR but for `./build/mvn` one has to add a profile for Hadoop 2/3 (e.g. `-Phadoop-3`). Otherwise the build fails due to missing AWS classes.

Reply:
@martin-g . Why do you add your irrelevant comments like that? Actually, it doesn't look like a good way to collaborate because this is open to all the community member and your comments are distracting the the other people from the PR content itself.

Reply:
I'd like to recommend you to file a JIRA officially instead and go for it. :)